doc/modules/model_persistence.rst

```diff
@@ -89,4 +88,28 @@ another architecture is not supported.
 
 If you want to know more about these issues and explore other possible
 serialization methods, please refer to this
-`talk by Alex Gaynor <https://pyvideo.org/video/2566/pickles-are-for-delis-not-software>`_.
+`talk by Alex Gaynor
+<https://pyvideo.org/video/2566/pickles-are-for-delis-not-software>`_.
+
+Interoperable formats
+---------------------
+
+For production and quality control needs, exporting the model in `Predictive
+Model Markup Language (PMML)
+<http://dmg.org/pmml/v4-4-1/GeneralStructure.html>`_ or `Open Neural Network
```

Comment:
Maybe let's put ONNX first, I haven't yet met people using PMML in practice :)

Reply:
We should probably mention the license as it's significantly more restrictive than scikit-learn's and people might not expect that.

Reply:
Done. Also, citing sklearn2pmml for PMML conversion, made me link sklearn-onnx for ONNX conversion.